agent/06_agent_create_cluster.sh

```diff
@@ -12,6 +12,7 @@ source $SCRIPTDIR/validation.sh
 source $SCRIPTDIR/release_info.sh
 source $SCRIPTDIR/agent/common.sh
 source $SCRIPTDIR/agent/iscsi_utils.sh
+source $SCRIPTDIR/agent/e2e/agent-tui/utils.sh
```

Comment:
In run_agent_test_cases(), bad_dns test case should not run if AGENT_E2E_TEST_BOOT_MODE=ISO_NO_REGISTRY.

Reply:
yes, we will do `exit 0` as soon as assiseted install UI is up and none of the other actions/tests needs to be done for OVE case for now